src/App.js

```diff
@@ -918,6 +925,9 @@ export default class App extends Component {
               //console.log("DAPPARATUS UPDATE",state)
 
               const { account, favorites } = state;
+
+              console.log('ACCOUNT ADDRESS:', account);
```

Comment:
Is it really needed here?

Reply:
I was asked to add some logging about user current address. This was later moved to menu. So now we can remove this console statement.